tests/beaker/tests/create_lv_with_param_size_is_minsize.rb

```diff
@@ -0,0 +1,55 @@
+require 'master_manipulator'
+require 'lvm_helper'
+require 'securerandom'
+
+test_name "FM-4579 - C96585 - create logical volume with parameter 'size'"
+test_name "FM-4579 - C96586 - create logical volume with parameter 'size_is_minsize'"
```

Comment:
Avoid having multiple tests in a single test file. If one test can represent a class of tests then you should only use that single test. If the tests are unique and valuable then the tests should be in separate files.